### PR TITLE
Add document for VDDK log level configuration in troubleshooting.md

### LINKED ISSUE #362: Document how to change VDDK logging level
**Describe the bug**
We should document how to change VDDK logging level here: https://github.com/vmware-tanzu/velero-plugin-for-vsphere/blob/main/docs/troubleshooting.md

**To Reproduce**

[Steps to reproduce the behavior]

**Expected behavior**

[A clear and concise description of what you expected to happen]

**Troubleshooting Information**

[Please refer to the [Troubleshooting](https://github.com/vmware-tanzu/velero-plugin-for-vsphere/blob/main/docs/troubleshooting.md) page and collect the required information]

**Screenshots**

[If applicable, add screenshots to help explain your problem.]

**Anything else you would like to add:**

[Miscellaneous information that will assist in solving the issue]

### PR DESCRIPTION
**Pull request by xinyanw409**
Signed-off-by: xinyanw409 <wxinyan@vmware.com>

**What this PR does / why we need it**:
Add documentation for vddk log level configuration.
**Which issue(s) this PR fixes**:

Fixes #362 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -48,3 +48,37 @@ Apart from logs in the general case, extra logs might also be optionally expecte
...
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  # any name can be used; Velero uses the labels (below)
+  # to identify it rather than the name
+  name: vddk-config
+  # must be in the velero namespace
+  namespace: velero
+  labels:
+    # this value-less label identifies the ConfigMap as
```
Comment (xing-yang) [inline docs/troubleshooting.md:66]: which label is "value-less"?

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/564/

File: docs/troubleshooting.md (hunk 1 above)
Comment (xinyanw409) [reply docs/troubleshooting.md]: Fix the incorrect comments.

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/565/

**Review by xing-yang — APPROVED** [review]
LGTM

**Pull request merged**